packages/fxa-payments-server/jest.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  'roots': [
```

Comment:
Decided to give in and try using the Jest test runner that comes with Create React App. It comes with a lot of things out of the box and seems to be getting more popular for React things.